config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
`git rm --cached config/database.yml `
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

Reply:
how to create the new file?

Reply:
fatal: pathspec 'config/database.yml.example' did not match any files
i keep getting this error when i try to add the new file
what should i do to solve this problem?

Reply:
Steps:
- Create new file named `database.yml.example` in `config/` folder with the same content `database.yml`
- gitignore `config/database.yml` file
- `git add config/database.yml.example`

Reply:
i able to add the new file, thank you for the advise